setup-dev.sh

```diff
@@ -5,6 +5,25 @@ echo "# Install testing tools."
 echo "# Only works with Ubuntu / APT."
 echo "# --------------------------------------"
 
+hook_script_name=".git/hooks/pre-push"
+
+if [ -t 1 ]; then
+    while true; do
+        echo "Install a git hook to automatically lint files before pushing? ([y]/N)"
```

Comment:
The convention with `y/n` prompts is usually that the capital letter is the default (but the user can still enter either option in any case). In this case we want the `N` to be the default, so we can change this to `y/N`.